webapp/src/pages/Map.tsx

```diff
@@ -1522,14 +1547,28 @@ function MapPage() {
             </div>
             <div id="sliderContainer" style={{ margin: '5px 10px' }}>
               <Slider
-                allowCross={false}
-                included={false}
                 min={0}
                 max={geoMsgFilterMaxOffset}
                 value={filterOffset}
-                onChange={(e) => {
-                  dispatch(setGeoMsgFilterOffset(e as number))
+                onChange={(_, value) => {
```

Comment:
suggestion(non-blocking): Define the types explicitly to leverage the typescript compiler (and the resulting intellisense) for this code to make it more maintainable and self-describing.
```suggestion
                onChange={(_: Event, value: number | number[]) => {
```